app/src/main/java/com/worldsnas/starplayer/view/PlayerFragment.kt

```diff
@@ -32,5 +45,34 @@ class PlayerFragment : Fragment() {
             .appComponent((activity!!.application as App).appComponent).build()
         playerComponent.inject(this)
 
+        initializeExoplayer()
+    }
+
+    private fun initializeExoplayer() {
+        exoPlayer = SimpleExoPlayer.Builder(context!!).build()
+        viewBinding.exoControlView.player = exoPlayer
+
+        val mediaSource = buildMediaSource(Uri.parse("asset:///take_your_time.mp3"))
+        exoPlayer.playWhenReady = playWhenReady
+        exoPlayer.seekTo(currentWindow, playbackPosition)
+        exoPlayer.prepare(mediaSource, false, false)
+    }
+
+    private fun buildMediaSource(uri: Uri): MediaSource {
+        val dataSourceFactory = DefaultDataSourceFactory(context, "Star-player")
+        return ProgressiveMediaSource.Factory(dataSourceFactory)
+            .createMediaSource(uri)
+    }
+
+    private fun releaseExoPlayer() {
+        playWhenReady = exoPlayer.playWhenReady
```

Comment:
intori negah dashtan maghadir fayde i nadare
be nazaram felan p
akesh konin behtar bashe